triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -790,6 +838,15 @@ object Runner extends StrictLogging {
   val maxParallelSubmissionsPerTrigger = 8
   val maxTriesWhenOverloaded = 6
 
+  /** Maximum number of in-flight commands that we shall allow *before* the ledger
+    *  client automatically fails submit requests
+    */
+  val maxInFlightCommands = 42
```

Comment:
Does anyone have any sensible values for this (its currently guessed!)?

Reply:
The community edition start to back pressure at 100. 
so between 20 and 50 max.

Anyway this should be configurable.

Reply:
Added in a `trigger-config.conf` file to trigger resources. using `pureconfig` we can now configure the trigger CLI/service on launch using either:
- `java -Dtrigger.parallelism=2 ...`
- or, `TRIGGER_MAX_INFLIGHT_COMMANDS=100; java ...`

This should also enable easier testing (in their) of any throttling or in-flight rate limiting behaviour that we have.

Reply:
As discussed offline, I'll remove this config code for now and add it into a separate PR

Reply:
Set this value to be 50 (the upper value in the range suggested).

Reply:
PR for trigger runner configuration is https://github.com/digital-asset/daml/pull/15482